dataline-workers/src/main/java/io/dataline/workers/process/DockerProcessBuilderFactory.java

```diff
@@ -63,7 +64,7 @@ public ProcessBuilder create(final Path jobRoot, final String imageName, final S
             imageName);
     cmd.addAll(Arrays.asList(args));
 
-    LOGGER.debug("Preparing command: {}", cmd);
+    LOGGER.debug("Preparing command: {}", Joiner.on(" ").join(cmd));
```

Comment:
Changed the logging so it is possible to copy & paste the the command in a console